slp_abacus/slp_abacus/parse/transformers/parent_calculator_transformer.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from otm.otm.entity.parent_type import ParentType
+from slp_abacus.slp_abacus.objects.diagram_objects import DiagramComponent, DiagramTrustZone
+from slp_abacus.slp_abacus.parse.transformers.transformer import Transformer
+
+PARENT_TYPES = {
+    DiagramComponent: ParentType.COMPONENT,
+    DiagramTrustZone: ParentType.TRUST_ZONE
+}
+
+
+def get_parent_type(element) -> Optional[ParentType]:
+    try:
+        return PARENT_TYPES[element.__class__]
+    except KeyError:
+        return None
+
+
+class ParentCalculatorTransformer(Transformer):
```

Comment:
This class is not being used because we are not managing parent relationships as for now, so we prefer to remove it.